### PR TITLE
Story/graph ui enhancements/488

### PR DESCRIPTION
**Pull request by andrea-williams**
- Added different symbols to Morecast graph
- Modified colours used in graph for better contrast and colour blindness compatibility

### CODE REVIEW

File: src/api/observationAPI.ts [hunk 1]
```diff
@@ -14,18 +14,18 @@ export interface ReadingValue {
...
 }
 
-export interface ReadingsResponse {
-  hourlies: Reading[]
+export interface ObservationsResponse {
+  hourlies: Observation[]
 }
 
-export async function getReadings(stationCodes: number[]): Promise<Reading[]> {
+export async function getObservations(stationCodes: number[]): Promise<Observation[]> {
```
Comment (Sybrand) [inline src/api/observationAPI.ts:26]: I've added onto the api re-factor ticket to confirm with this: https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/268

File: src/features/fireWeather/components/graphs/WxDataGraph.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ interface Props {
 }
 
 const WxDataGraph = ({
-  readingValues = [],
+  observedValues: observedValues = [],
```
Comment (Kyubinhan) [inline src/features/fireWeather/components/graphs/WxDataGraph.tsx:34]:
```suggestion
  observedValues = [],
```

File: src/features/fireWeather/components/graphs/TempRHGraph.tsx [hunk 3]
```diff
@@ -245,7 +245,7 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
         .range([0, width])
       const yTempScale = d3
         .scaleLinear()
-        .domain([-10, 45])
+        .domain([-10, 40])
```
Comment (Kyubinhan) [inline src/features/fireWeather/components/graphs/TempRHGraph.tsx:248]: Line 531 also needs to be changed
svg.append('g').call(d3.axisLeft(yTempScale).tickValues([-10, 5, 20, 35, 40]))

File: src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx [hunk 4]
```diff
@@ -39,7 +39,7 @@ interface Props {
 const WxDataToggles = ({
   toggleValues,
   setToggleValues,
-  noReadings,
+  noObservations: noObservations,
```
Comment (Kyubinhan) [inline src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx:42]:
```suggestion
  noObservations,
```

File: src/features/fireWeather/components/graphs/TempRHGraph.tsx [hunk 5]
```diff
@@ -647,24 +669,49 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
       })
       d3Utils.addLegend({
         svg,
-        text: 'High Res Model Temp',
-        color: styles.highResModelTempDotColor,
-        fill: 'none',
+        text: 'HRDPS Temp',
```
Comment (Kyubinhan) [inline src/features/fireWeather/components/graphs/TempRHGraph.tsx:672]: Can you update the text for HRDPS in the tooltip as well?

**Review by Kyubinhan — CHANGES_REQUESTED** [review]
Overall looks great,
I have some minor requesting changes

**Review by Sybrand — APPROVED** [review]
Looks good - just address the comments from Kybin.

**Review by Kyubinhan — APPROVED** [review]

**Pull request merged**